docs/tutorials/cli.mdx

```diff
@@ -0,0 +1,178 @@
+
+```sh
+svix application create --data-name "My Application"
+```
+
+This command creates a new application on Svix and returns the raw application object as JSON.
+
+## List your applications
+
+Now that we've created an app we can list our existing apps with the following command:
+
+```sh
+svix application list
+```
+
+The list command by default returns the first 50 applications, you can change this limit by adding the `--limit` flag
+
+```sh
+svix application list --limit 100 # get the first 100 applications
+```
+
+## Add an endpoint
+
+Next let's add an endpoint to the application we created above.  The create command should have returned an Application ID.
+Since the application name is not unique, will need its ID to make changes.
+
+Let's add a webhook endpoint with the url `http://example.com/webhook`. Every endpoint must have a version associated with it
+to help you version your webhook API.  For this example we will set the version to `1`.
+
+```sh
+svix endpoint create <MY-APP-ID> --data-url http://example.com/webhook --data-version 1
+```
+
+## Send a message
+
+Now that we have an application created and an endpoint configured to listen to messages from that app, we are ready to send our first
+webhook message! Let's send a `user.created` event with the user's username and email.
+
+```sh
+svix message create <MY-APP-ID> --data-eventType "user.created" --data-payload "{\"username\":\"new_user\",\"email\":\"new_user@example.com\"}"
+```
+
+This message will now be sent to Svix and dispatched to all webhook endpoints listening to your application.
+
+## Get creative
+
+The Svix CLI has commands to interact with every part of the Svix API.
+
+To get a complete list of commands, run `svix --help`.
+
+Each command accepts raw JSON input as either the first positional argument or piped in over `stdin`.
+
+With this in mind, you can chain Svix CLI commands together with other command line tools like the popular JSON parser
+[jq](https://stedolan.github.io/jq/) to create powerful snippets to improve your workflow.
+
+For example, to automatically open the dashboard for the app we created in step 3, on macOS you can run the following command:
+
+
+```sh
+svix application get <MY-APP-ID> | jq '.id' | xargs svix auth dashboard | jq '.url' | xargs open
+```
+
+With a slight modification, we can convert this one-liner to a shell function:
+
+```zsh
+function dashboard {
+   svix application get "$1" | jq '.id' | xargs svix auth dashboard | jq '.url' | xargs open
+}
+```
+
+Which you can run like so:
+
+```zsh
+dashboard <MY-APP-ID>
+```
+
+## Final note
+
+You can find the source for the Svix CLI on our [Github](https://github.com/svix/svix-cli). If you find a bug or have an idea
+on how it can be improved, we encourage you to file an [issue](https://github.com/svix/svix-cli/issues/new) or open a
+[PR](https://github.com/svix/svix-cli/pulls). We look forward to your feedback!
```

Comment:
This feels like a great final note for a blog post, but not so much for a tutorial on a docs website. I almost think like this would just be better without this closing section at all. The same way we don't have "final note" on any other doc page.

Reply:
removed